package.json

```diff
@@ -9,15 +9,16 @@
     "registry": "node ./dist/registry.js",
     "compile": "npx tsc",
     "test": "jest --verbose --coverage --no-cache",
-    "bench": "node benchmarking/bench.js",
+    "benchERC20": "node benchmarking/benchERC20.js",
+    "benchMarketplace": "node benchmarking/benchMarketPlace.js",
```

Comment:
Let's make this just `bench` and `benchERC20`. In general, there is no marketplace protocol: Swap supports multiple token kinds, SwapERC20 supports ERC20 only.